addons/binding/org.openhab.binding.vallox/README.md

```diff
@@ -0,0 +1,143 @@
+# Vallox Binding
+
+Interface to a vallox central venting unit.
+
+Tested with 
+* Vallox KWL 90 SE and 
+* USR-TCP232-410 RS485-to-ethernet interface
+
+## Prerequisite
+
+This Openhab2 bundle so far only works with a RS485-to-ethernet gateway.
+It does not support a local serial interface of the device executing OH2.
+
+I do not have any experience with direct serial RS485 communication under Java and
+do not have any device for testing it. So if you can contribute any code for direct
+RS485 serial write and read, I might easily merge that.
+
+## Configuration
+
+Configure the serial interface such that it acts as TCP server on some port of your choice. 
+Configure the serial side as 
+* baudrate: 9600
+* data size: 8 bit
+* parity: none
+* stop bits: 1 
+
+## Install
```

Comment:
Please remove this section - this is imho nothing special (and should not be special) for this binding, but the normal way of configuring the system.